src/routes/v0.ts

```diff
@@ -199,7 +199,8 @@ export default async function (
 
           // 2) Add annual savings from pregenerated model
           estimated_annual_savings:
-            IRA_STATE_SAVINGS[location.state].estimated_savings_heat_pump_ev,
+            IRA_STATE_SAVINGS[location.state]?.estimated_savings_heat_pump_ev ??
+            IRA_STATE_SAVINGS['US'].estimated_savings_heat_pump_ev,
```

Comment:
Nit: Is this fallback data? Should we store this as CONST somewhere to make that more expressive? IE: `const DEFAULT_HEAT_PUMP_EV_SAVINGS = ...`?